### PR TITLE
Ember changelog, refs: #379

### PR DESCRIPTION
**Pull request by luca-bruegger**

### CODE REVIEW

File: bin/prepare-frontend.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+cp CHANGELOG.md frontend/markdown/
```
Comment (TheWalkingLeek) [inline bin/prepare-frontend.sh:2]: Do we need to remove the symlink before copying or does this override?

File: frontend/app/components/changelog.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import Component from '@ember/component';
```
Comment (TheWalkingLeek) [inline frontend/app/components/changelog.js:1]:
```suggestion
import Component from '@glimmer/component';
```
ember octane prefers glimmer :)

File: frontend/app/templates/components/footer.hbs [hunk 3]
```diff
@@ -1,6 +1,12 @@
+{{#if this.isChangelogOpen}}
+  <Changelog @onAbort={{this.toggleChangelogModal}} @title="Changelog"/>
```
Comment (TheWalkingLeek) [inline frontend/app/templates/components/footer.hbs:2]: We don't need to translate the title here right? Probably fine

File: frontend/package.json [hunk 4]
```diff
@@ -61,6 +62,7 @@
     "ember-intl": "^4.4.0",
     "ember-islands": "^1.5.1",
     "ember-load-initializers": "^2.1.2",
+    "ember-markdown-html": "^0.0.1",
```
Comment (TheWalkingLeek) [inline frontend/package.json:65]: We don't need both addons right? We're using the showdown addon right? Looks better

File: public/frontend-index.html [hunk 5]
```diff
@@ -1 +0,0 @@
-../frontend/dist/index.html
```
Comment (TheWalkingLeek) [inline public/frontend-index.html:1]: Please run frontend:clean before pushing

**Review by TheWalkingLeek — CHANGES_REQUESTED** [review]
Gooooood stuff, small small notes

File: frontend/app/templates/components/footer.hbs (hunk 3 above)
Comment (luca-bruegger) [reply frontend/app/templates/components/footer.hbs:2]: It is the same in every language.

File: bin/prepare-frontend.sh (hunk 1 above)
Comment (luca-bruegger) [reply bin/prepare-frontend.sh:2]: cp, by default, overwrites.

**Pull request merged**